nncf/quantization/algorithms/bias_correction/algorithm.py

```diff
@@ -376,25 +377,23 @@ def _compute_bias_shift(
         for feed_dict in feed_dicts:
             q_output = engine.infer(feed_dict)
             q_output = self._backend_entity.process_model_output(q_output, output_tensor_name)
-            q_outputs.append(self._backend_entity.tensor_processor.mean_per_channel(q_output, channel_axis).tensor)
+            q_outputs.append(mean_per_channel(q_output, channel_axis))
         # Here we get the per-sample average, so the axis is 0.
-        q_output = np.mean(q_outputs, axis=0)
-        return output_fp - q_output
+        q_output = fns.mean(fns.stack(q_outputs), axis=0)
+        return fns.stack(output_fp) - q_output
 
     @staticmethod
-    def _get_bias_shift_magnitude(current_bias_value: np.ndarray, updated_bias_value: np.ndarray) -> float:
+    def _get_bias_shift_magnitude(current_bias_value: Tensor, updated_bias_value: Tensor) -> float:
```

Comment:
Isn't the resulting value the `Tensor` now?

Reply:
fixed